javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,50 @@
+const width = 22;
+const height = 3;
+const area = width / height;
+
+console.log(`Value of area: ${area}`);
+console.log(`typeof area: ${typeof area}`);
+const bill = 200;
+const payment = 300;
+const change = payment - bill;
+
+console.log(`Value of change: ${change}`);
+console.log(`typeof change: ${typeof change}`);
+
+const quizzes = 12;
+const midterm = 21;
+const final = 33;
+
+const grade = quizzes / midterm / final;
```

Comment:
These should be getting added together, and then you would divide the result of that by 3